runtime/bifrost-kusama/src/lib.rs

```diff
@@ -1005,11 +1014,11 @@ impl pallet_xcm::Config for Runtime {
 	type LocationInverter = LocationInverter<Ancestry>;
 	type SendXcmOrigin = EnsureXcmOrigin<Origin, LocalOriginToLocation>;
 	type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;
-	type XcmExecuteFilter = Everything;
+	type XcmExecuteFilter = Nothing;
```

Comment:
Nothing will deny all Xcm transaction to Relaychain. A custom filter would be better.

Reply:
only affect `execute` in `PolkadotXcmPallet` and we do not rely on this to send any crosschain message, for safety better disable